Test/Unit/Block/JsTest.php

```diff
@@ -0,0 +1,222 @@
+    protected $configHelper;
+    /**
+     * @var \Magento\Framework\App\Helper\Context
+     */
+    protected $helperContextMock;
+    /**
+     * @var \Magento\Framework\View\Element\Template\Context
+     */
+    protected $contextMock;
+
+    /**
+     * @var BlockJs
+     */
+    protected $block;
+
+    /**
+     * @inheritdoc
+     */
+    protected function setUp()
+    {
+        $this->helperContextMock = $this->createMock(\Magento\Framework\App\Helper\Context::class);
+        $this->contextMock = $this->createMock(\Magento\Framework\View\Element\Template\Context::class);
+
+        $methods = [
+            'isSandboxModeSet', 'isActive', 'getAnyPublishableKey',
+            'getPublishableKeyPayment', 'getPublishableKeyCheckout',
+            'getReplaceSelectors', 'getGlobalCSS', 'getPrefetchShipping'
+        ];
+
+        $this->configHelper = $this->getMockBuilder(HelperConfig::class)
+            ->setMethods($methods)
+            ->setConstructorArgs(
+                [
+                    $this->helperContextMock,
+                    $this->createMock(\Magento\Framework\Encryption\EncryptorInterface::class),
+                    $this->createMock(\Magento\Framework\Module\ResourceInterface::class),
+                    $this->createMock(\Magento\Framework\App\ProductMetadataInterface::class)
+                ]
+            )
+            ->getMock();
+
+        $this->block = $this->getMockBuilder(BlockJs::class)
+            ->setMethods(['configHelper', 'getUrl'])
+            ->setConstructorArgs(
+                [
+                    $this->contextMock,
+                    $this->configHelper,
+                ]
+            )
+            ->getMock();
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function testGetTrackJsUrl()
+    {
+        $file = 'track.js';
+
+        // For CDN URL in sandbox mode
+        $this->setSandboxMode();
+        $result = $this->block->getTrackJsUrl();
+        $expectedUrl = HelperConfig::CDN_URL_SANDBOX . DIRECTORY_SEPARATOR . $file;
+
+        $this->assertEquals($expectedUrl, $result, 'Not equal CDN Url in Sandbox mode');
+
+        // ReInit for production mode.
+        $this->setUp();
+
+        // For CDN URL in production mode.
+        $this->setSandboxMode(false);
+        $result = $this->block->getTrackJsUrl();
+        $expectedUrl = HelperConfig::CDN_URL_PRODUCTION . DIRECTORY_SEPARATOR . $file;
+
+        $this->assertEquals($result, $expectedUrl, 'Not equal CDN Url in Production mode');
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function testGetConnectJsUrl()
+    {
+        $file = 'connect.js';
+
+        // For CDN URL in sandbox mode
+        $this->setSandboxMode();
+        $result = $this->block->getConnectJsUrl();
+        $expectedUrl = HelperConfig::CDN_URL_SANDBOX . DIRECTORY_SEPARATOR . $file;
+        $this->assertEquals($result, $expectedUrl, 'Not equal CDN Url in Sandbox mode');
+
+        // ReInit for production mode.
```

Comment:
this test is testing two different things. can you split into two different methods?